newsubprojects.md

```diff
@@ -0,0 +1,183 @@
+# New Subproject Process
+
+Project Jupyter is organized as a set of Subprojects that are each a GitHub repository with a
+development team that follows the Jupyter governance, license and contribution model. Officially supported and maintained subprojects are hosted on the [jupyter](https://github.com/jupyter) GitHub orgnization. This document describes the process by which new Subprojects are created in or moved to the Jupyter organization.
+
+There are two ways that new Subprojects are created:
+
+1. Direct subproject creation.
+2. Incorporation of an existing, external subproject.
+
+
+## Criteria for official subprojects
+
+The following criteria will be used in evaluating Subprojects for incorporation into the official
+Jupyter organization. These apply to any Subproject, regardless of who is developing the
+Subproject. Subprojects should:
+
+* Have an active developer community that offers a sustainable model for future development.
+* Have an active user community.
+* Use solid software engineering with documentation and tests hosted with appropriate
+  technologies ([Read The Docs](https://readthedocs.org/), [Travis](https://travis-ci.org/)
+  are examples of technologies that can be used).
+* Demonstrate continued growth and development.
+* Integrate well with other official Subprojects.
+* Be developed according to the Jupyter governance and contribution model that is documented
+  [here](https://github.com/jupyter/governance).
+* Have a well-defined scope.
+* Be well packaged using appropriate technologies such as pip, conda, npm, bower, docker, etc.
+* Not canibalize or fork an existing official subproject.
+
+The most important question in evaluating subprojects for incorporation as official Subprojects
+is this: is there broad agreement in the community and Steering Council that we are going to
+develop and maintain the Subproject in an official capacity?
+
+The two paths for official Subproject creation are now detailed.
+
+
+## Direct subproject creation
+
+In some cases, Steering Council members can immediately create new official Subprojects in the main
+Jupyter organization. This option will be used when the new Subproject is clearly part of the
+existing scope, activities and development of Project Jupyter. The process for direct Subproject
+creation is minimal and informal: Steering Council members should be in consensus about the
+Subproject's creation and notify the main Jupyter list of its creation.
+
+
+## Incorporation of an existing, external subproject
+
+In other cases, Subprojects proposed for incorporation will have existed for some time outside the
+official Jupyter organization. This section describes the incorporation process for these existing,
+external subprojects. This process applies to Subprojects that have incubated under the
+`jupyter-incubatror` GitHub organization (see below), Subprojects developed under other GitHub
+organizations or even Subprojects developed in other public version control systems.
```

Comment:
I think it needs to be public, at least during the review process described here, so that people can actually review it.